packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -676,3 +677,9 @@ export interface IndexerLeaderboardContest {
 export interface GetIndexerLeaderboardContestsResponse {
   contests: IndexerLeaderboardContest[];
 }
+
+export interface GetIndexerVrtxTokenInfoParams {
+  tokenInfoType: IndexerVrtxTokenInfoType;
+}
+
+export type GetIndexerVrtxTokenInfoResponse = number;
```

Comment:
Is it a number or a string from backend?

Reply:
Is this in X18? ie is it 10^9+18 or just 10^9 being returned?

Reply:
it's just a number and it's 10^9

Reply:
Can we leave a JSDoc comment indicating that this does not include decimals?